examples/corb2-project/build.gradle

```diff
@@ -0,0 +1,111 @@
+/*
+ * This buildscript shows how corb dependencies can be easily declared and then uses in a Gradle configuration named
+ * "corb". This configuration is then used as the classpath for CorbTask, which is an extension of Gradle's
+ * JavaExec task that exposes the corb options as task attributes. The CorbTask will apply any corb options set via
+ * task attributes, project properties, or System properties.
+ */
+plugins {
+    id "com.marklogic.ml-gradle" version "2.0"
+}
+
+repositories {
+    jcenter()
+
+    // Needed for corb dependencies: XCC, Jasypt(optional)
+    maven { url "http://developer.marklogic.com/maven2/" }
+
+    // Needed to resolve CoRB2
+    maven { url "http://rjrudin.github.io/marklogic-java/releases" }
```

Comment:
You can actually drop this - I published 2.2.1 of corb2 to jcenter - https://bintray.com/rjrudin/maven/marklogic-corb/view